rjd_gfx_d3d11.h

```diff
@@ -631,6 +635,9 @@ struct rjd_result rjd_gfx_command_buffer_commit(struct rjd_gfx_context* context,
 	cmd_buffer_d3d11->deferred_context = NULL;
 	cmd_buffer_d3d11->render_target = NULL;
 
+    rjd_gfx_command_buffer_destroy_d3d11(context_d3d11, cmd_buffer->handle);
```

Comment:
Nice find! I knew there was one more memory leak but wasn't sure where it was... 

Would you mind changing this to directly erase the slot? i.e.:
`rjd_slotmap_erase(context->slotmap_command_buffers, cmd_buffer->handle);`
I think that makes the intention of this code a bit clearer.